cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -68,6 +68,7 @@ type StaticAutoscaler struct {
 	lastScaleUpTime         time.Time
 	lastScaleDownDeleteTime time.Time
 	lastScaleDownFailTime   time.Time
+	lastRemoveUnregister    time.Time
```

Comment:
The name of this variable is unclear on first reading to me. 

Something like `lastUnregisteredRemovalTime` would match the naming scheme of the existing variables used for similar purposes.